src/main/resources/api.yaml

```diff
@@ -1438,6 +1438,9 @@ definitions:
         type: array
         items:
           type: string
+      selfRegistrationAllowed:
+        type: boolean
+        description: Flag indicating where Self-service registration is enabled for this service. If omitted, defaults to false.
```

Comment:
Grammatical nit... "Flag indicating *whether* Self-service registration is enabled for this service. If omitted, defaults to false."

Reply:
@jburke-amido  are you OK with the grammar? don't want to change it again! 😄

Reply:
Ah, it's a typo we have to fix :)